From a520812e38cad3613b1b1a096cd0fc8ef7e420c7 Mon Sep 17 00:00:00 2001
From: xeniape <xenia.fischer@stackable.tech>
Date: Tue, 23 Sep 2025 09:43:41 +0200
Subject: Update FMPP version

This is because FMPP Maven Plugin depends on FMPP in version 0.9.14
which itself depends on a Freemarker version that has not been pinned.
Instead it specifies a range which resolves to a SNAPSHOT version
which we don't want.
---
 sql/pom.xml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/sql/pom.xml b/sql/pom.xml
index 1fe6cf3389..02db6e49fb 100644
--- a/sql/pom.xml
+++ b/sql/pom.xml
@@ -364,6 +364,13 @@
       <plugin>
         <groupId>com.googlecode.fmpp-maven-plugin</groupId>
         <artifactId>fmpp-maven-plugin</artifactId>
+        <dependencies>
+          <dependency>
+            <groupId>net.sourceforge.fmpp</groupId>
+            <artifactId>fmpp</artifactId>
+            <version>0.9.16</version>
+          </dependency>
+        </dependencies>
         <executions>
           <execution>
             <id>generate-fmpp-sources</id>
